fio/lib/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Also, test that the package works and compiles fine by creating a new TS node script and using the lib.